CONTRIBUTING.md

```diff
@@ -1,33 +1,48 @@
-    - [Other C++ Coding Style Suggestions](#other-c-coding-style-suggestions)
-      - [Throw and return are not functions](#throw-and-return-are-not-functions)
-      - [Always use pre-increment and pre-decrement operators when you have a choice](#always-use-pre-increment-and-pre-decrement-operators-when-you-have-a-choice)
-      - [Place pointer and reference symbols with the type](#place-pointer-and-reference-symbols-with-the-type)
-    - [Removing Methods](#removing-methods)
-
-Making a Pull Request
----------------------
-1. To help the people who review your pull request, you must include a detailed description of what changes or additions the pull request includes and what you have done to test and verify the changes. If there is an existing issue or set of issues that documents the problem the pull request is solving, you can reference it. But, please make sure detailed information about the commits is easily available.
+Ways to contribute
+------------------
+There are lots of ways to contribute to the OpenSim project, and people with widely varying skill sets can make meaningful contributions. Please don't think your contribution has to be momentous to be appreciated. See a typo? Tell us about it or fix it! Here are some contribution ideas:
+
+- Use OpenSim and let us know how you're using it by posting to the [OpenSim user forum](https://simtk.org/forums/viewforum.php?f=XX).  
+- Ask and/or answer questions on the forum.
+- File bug reports, documentation problems, feature requests, and developer discussion topics using the GitHub [Issue tracker](https://github.com/opensim-core/opensim/issues).
+- Submit GitHub Pull Requests providing new features, examples, or bug fixes to code or documentation (see below).
+- If our hard work has helped you with yours, please considering acknowledging your use of OpenSim and encourage others to do so. Please cite the following papers:
+
+    Delp SL, Anderson FC, Arnold AS, Loan P, Habib A, John CT, Guendelman E, Thelen DG (2007) OpenSim: open-source software to create and analyze dynamic simulations of movement. *IEEE Trans Biomed Eng* 54:1940–50
+
+    Seth A, Sherman M, Reinbolt JA, Delp SL (2011) OpenSim: A musculoskeletal modeling and simulation framework for in silico investigations and exchange. *Procedia IUTAM* 2:212–232
+
+
+
+Making a Pull Request (PR)
+--------------------------
+Please don't surprise us with big out-of-the-blue PRs. Preferably, target an existing Issue, post a comment to that effect in the Issue. That way you can engage in discussion with others to coordinate your work with theirs and avoid duplication, to see what people think of the approach you have planned, and so on. If there is not yet a relevant Issue in place, a great way to start is by creating one, and then engaging in an Issue conversation.
+
+If you have never made a PR before there are excellent GitHub tutorial to familiarize you with the concepts and steps like the [GitHub Bootcamp](https://github.com).
```

Comment:
tutorial -> tutorials
Link should point [here](https://help.github.com/categories/bootcamp/)